### ISSUE TITLE
Shell scripts to ease use

### ISSUE THREAD

**Issue by SirPython** [Cardshifter/Cardshifter#424]
Not every user who is going to playing the game will know to type in

    java -jar <snapshot>.jar

to run the client or the server (at least, that is how I run them in the distributed zip files). I think it would be helpful (and it might make things happen more quickly) if there was a shell script for running the client and the server.

<hr>

Another useful shell script might be one for automatically check for updates. The script would check the distributions page and, if a distribution is found that is from a later date than the one owned, it could prompt the user if they want to by it.

I imagine the flow would look like this:

> Checking for updates...
> Update found: version [...]
> Would you like to install? (y/n)
> Installing updates...

**Comment by jacwah** [comment]
The Java client is not maintained any longer, and most users will only interact with Cardshifter through their web browser. The only people who need to use the Java distribution are those who wish to host their own server. I think they can be trusted to have some amount of technical knowledge.

I know the JavaFX client could simply be double clicked to launch, though terminal for logging. Double clicking does not work for the server. A shell script or some other way to launch the server in a terminal window might be useful.

**Comment by jacwah** [comment]
As for your second suggestion, I think it could potentially be useful, though I'm not sure how much more inclined a server op would be to run a script once in a while than check news/releases pages.

It shouldn't be a shell script in any case, as we have to support Windows. If we choose to implement it, it could maybe be done in Groovy.

**Comment by Zomis** [comment]
I like the idea of automatic updates. As for the `java -jar`, let's remind ourselves that Minecraft server owners are able to use that. I don't see the need for any kind of script for that.

**Comment by Zomis** [comment]
However, speaking of automatic-updates. What if there has been changes to the mod files locally and there are also new versions in the update? I foresee problems...

**Comment by jacwah** [comment]
Indeed, if a user has installed a 3rd party mod or made their own, they wouldn't want to update before they knew they were compatible with the new version.
